wrappers/python/python.hpp

```diff
@@ -111,6 +116,79 @@ std::string matrix_to_string(const T(&arr)[N][M])
 #define BIND_RAW_ARRAY_PROPERTY(T, member, valueT, SIZE) #member, BIND_RAW_ARRAY_GETTER(T, member, valueT, SIZE), BIND_RAW_ARRAY_SETTER(T, member, valueT, SIZE)
 #define BIND_RAW_2D_ARRAY_PROPERTY(T, member, valueT, NROWS, NCOLS) #member, BIND_RAW_2D_ARRAY_GETTER(T, member, valueT, NROWS, NCOLS), BIND_RAW_2D_ARRAY_SETTER(T, member, valueT, NROWS, NCOLS)
 
+// Map format->data type for various things
+template <rs2_format> struct FmtToType { using type = uint8_t; }; // Default to uint8_t
+#define MAP_FMT_TO_TYPE(F, T) template <> struct FmtToType<F> { using type = T; }
+MAP_FMT_TO_TYPE(RS2_FORMAT_Z16, uint16_t);
+//MAP_FMT_TO_TYPE(RS2_FORMAT_DISPARITY16, );
+MAP_FMT_TO_TYPE(RS2_FORMAT_XYZ32F, float);
+MAP_FMT_TO_TYPE(RS2_FORMAT_YUYV, uint8_t);
+MAP_FMT_TO_TYPE(RS2_FORMAT_RGB8, uint8_t);
+MAP_FMT_TO_TYPE(RS2_FORMAT_BGR8, uint8_t);
+MAP_FMT_TO_TYPE(RS2_FORMAT_RGBA8, uint8_t);
+MAP_FMT_TO_TYPE(RS2_FORMAT_BGRA8, uint8_t);
+MAP_FMT_TO_TYPE(RS2_FORMAT_Y8, uint8_t);
+MAP_FMT_TO_TYPE(RS2_FORMAT_Y16, uint16_t);
+//MAP_FMT_TO_TYPE(RS2_FORMAT_RAW10, );
+MAP_FMT_TO_TYPE(RS2_FORMAT_RAW16, uint16_t);
+MAP_FMT_TO_TYPE(RS2_FORMAT_RAW8, uint8_t);
+MAP_FMT_TO_TYPE(RS2_FORMAT_UYVY, uint8_t);
+//MAP_FMT_TO_TYPE(RS2_FORMAT_MOTION_RAW, );
+MAP_FMT_TO_TYPE(RS2_FORMAT_MOTION_XYZ32F, float);
+//MAP_FMT_TO_TYPE(RS2_FORMAT_GPIO_RAW, );
+//MAP_FMT_TO_TYPE(RS2_FORMAT_6DOF, );
+MAP_FMT_TO_TYPE(RS2_FORMAT_DISPARITY32, float);
+MAP_FMT_TO_TYPE(RS2_FORMAT_Y10BPACK, uint16_t);
```

Comment:
This format has a packing of 4 pixels in 5 bytes. probably should be treated with uint8_t